hello_microverse.html

```diff
@@ -0,0 +1,13 @@
+﻿<!DOCTYPE html>
+
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Hello Microverse</title>
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
+</body>
+</html>
```

Comment:
![image](https://github.com/Movgto/Hello_Microverse/assets/78251927/0ffcf6a6-bf73-4195-9767-200146f22a99)

Please kindly read through the errors to fix them 💪